.azure/azure-pipelines.ci.yml

```
@@ -224,6 +224,14 @@ stages:
       tls: stub
       extraBuildArgs: -SanitizeAddress
       config: Debug
+  - template: ./templates/build-config-user.yml
+    parameters:
+      image: ubuntu-latest
+      platform: linux
+      arch: x64
+      tls: openssl
+      extraBuildArgs: -CodeCheck
+      config: Debug
```

You need to use the `extraName` parameter to make the job name unique, otherwise AZP complains.
```suggestion
      config: Debug
      extraName: 'codecheck'
```